src/cryptoadvance/specter/server_endpoints/wallets.py

```diff
@@ -1323,44 +1323,116 @@ def sort(tx):
     return {"txlist": json.dumps(txlist), "pageCount": page_count}
 
 
-@wallets_endpoint.route("/wallet/<wallet_alias>/addresses/", methods=["GET"])
+def process_addresses_list(
+    addresses_list, idx=0, limit=100, sortby=None, sortdir="asc"
```

Comment:
Can you add a function-documentation here?

Reply:
No problem. It has been added.